conf/testnet/bitcoin.conf

```diff
@@ -0,0 +1,4 @@
+rpcuser=stacks
```

Comment:
this file isn't needed, only the sample file. if this file doesn't exist locally, the script should copy sample to this one (and added to gitignore)

Reply:
That is correct. I must have uploaded it without realizing before adding it to gitignore. (it's already in gitignore). I'll delete the file in a new commit.